### PR TITLE
fix: Update to Tiled 1.1.5

### PR DESCRIPTION
**Pull request by RobLoach**
https://github.com/bjorn/tiled/releases/tag/v1.1.5

### CODE REVIEW

File: org.mapeditor.Tiled.json [hunk 1]
```diff
@@ -40,8 +40,8 @@
       "sources": [
         {
           "type": "archive",
-          "url": "https://github.com/bjorn/tiled/archive/v1.1.3.tar.gz",
-          "sha256": "13a4861b9c0f68936ee1f4aa463674bb4729b2574946934994d6fac3b6a17d21"
+          "url": "https://github.com/bjorn/tiled	/archive/v1.1.5.tar.gz",
```
Comment (nedrichards) [inline org.mapeditor.Tiled.json]: this space *looks* wrong, but doesn't seem to cause an actual issue downloading.

**Comment by nedrichards** [comment]
bot, build

File: org.mapeditor.Tiled.json (hunk 1 above)
Comment (RobLoach) [reply org.mapeditor.Tiled.json]: Oops. Thanks.

**Comment by nedrichards** [comment]
bot, build

**Comment by nedrichards** [comment]
build succeeded https://flathub.org/builds/#/builders/2/builds/1133

**Pull request merged**